device/vr/android/gvr/gvr_device.cc

```diff
@@ -233,8 +233,8 @@ std::vector<mojom::VRHitPtr> GvrDevice::HitTest(float x, float y)
   return hits;
 }
 
-std::vector<mojom::VRPlanePtr> GvrDevice::GetPlanes() {
-  std::vector<mojom::VRPlanePtr> planes;
+mojom::VRPlaneDeltasPtr GvrDevice::GetPlaneDeltas() {
+  mojom::VRPlaneDeltasPtr planes;
```

Comment:
I am not sure if this pointer is initialized to nullptr. I guess so but maybe making it explicit would be better?

Reply:
done